### PR TITLE
Add missing descriptions to account types

### PR DESCRIPTION
**Pull request by fowczarek**
I want to merge this change because of adding missing descriptions to account types.

Part of #13142

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migrations are either absent or optimized for zero downtime
* [ ] The changes are covered by test cases

### CODE REVIEW

**Review by SzymJ — APPROVED** [review]

File: saleor/graphql/account/types.py [hunk 1]
```diff
@@ -82,19 +82,35 @@ class AddressInput(BaseInputObjectType):
...
-    postal_code = graphene.String(required=True)
+    id = graphene.GlobalID(required=True, description="The ID of the address.")
+    first_name = graphene.String(
+        required=True, description="The given name of the address."
+    )
+    last_name = graphene.String(
+        required=True, description="The family name of the address."
+    )
+    company_name = graphene.String(
+        required=True, description="Company or organization."
```
Comment (IKarbowiak) [inline saleor/graphql/account/types.py:93]:
```suggestion
        required=True, description="Company or organization name."
```

File: saleor/graphql/account/types.py [hunk 2]
```diff
@@ -82,19 +82,35 @@ class AddressInput(BaseInputObjectType):
...
+    )
+    street_address_1 = graphene.String(
+        required=True, description="The first line of the address."
+    )
+    street_address_2 = graphene.String(
+        required=True, description="The second line of the address."
+    )
+    city = graphene.String(required=True, description="The city of the address.")
+    city_area = graphene.String(
+        required=True, description="The district of the address"
```
Comment (IKarbowiak) [inline saleor/graphql/account/types.py:103]:
```suggestion
        required=True, description="The district of the address."
```

File: saleor/graphql/account/types.py [hunk 3]
```diff
@@ -82,19 +82,35 @@ class AddressInput(BaseInputObjectType):
...
+    )
+    postal_code = graphene.String(
+        required=True, description="The postal code of the address."
+    )
     country = graphene.Field(
-        CountryDisplay, required=True, description="Shop's default country."
+        CountryDisplay, required=True, description="The count of the address."
```
Comment (IKarbowiak) [inline saleor/graphql/account/types.py:109]:
```suggestion
        CountryDisplay, required=True, description="The country of the address."
````

File: saleor/graphql/account/types.py [hunk 4]
```diff
@@ -82,19 +82,35 @@ class AddressInput(BaseInputObjectType):
...
+    )
+    country_area = graphene.String(
+        required=True, description="The country area of the address,"
```
Comment (IKarbowiak) [inline saleor/graphql/account/types.py:112]:
```suggestion
        required=True, description="The country area of the address."
```

**Review by IKarbowiak — APPROVED** [review]

File: saleor/graphql/account/types.py [hunk 5]
```diff
@@ -82,19 +82,35 @@ class AddressInput(BaseInputObjectType):
...
+    )
     country = graphene.Field(
-        CountryDisplay, required=True, description="Shop's default country."
+        CountryDisplay, required=True, description="The country of the address."
+    )
+    country_area = graphene.String(
+        required=True, description="The country area of the address."
     )
-    country_area = graphene.String(required=True)
     phone = graphene.String()
```
Comment (maarcingebala) [inline saleor/graphql/account/types.py:114]: Missing `phone` description; we could add it and mention that phone numbers are validated with Google's [libphonenumber](https://github.com/google/libphonenumber) library and provide a link to it. We use a package this is a Python port of the Google's one. People often ask about Saleor phone validation rules so I guess we should look through all `phone` fields and add a similar description.

**Pull request merged**